.gitignore

```diff
@@ -15,6 +15,7 @@ experiments
 coverage
 test-results.xml
 .history
+.fauna
```

Comment:
do the tests create a .fauna dir? if so, can they be mocked? if not, what's .fauna for (in this directory)?

Reply:
yes that's from the tests, https://github.com/fauna/fauna-shell/blob/c35346e2bbda5ef48c44a677d7713bb9a385f8bd/src/lib/file-util.mjs#L114

I'll mock that and get rid of this ignore